qiime_studio/server.py

```diff
@@ -13,6 +13,14 @@
 studio.after_request(add_cors_headers)
 
 
+# No output is visible, as for some reason it is held back
+# by the server flushed at seemingly random intervals
+# also, what should be the stdout, is being sent through stderr??
+def flush_stdio():
+    sys.stdout.flush()
+    sys.stderr.flush()
+
```

Comment:
How about [this flag?](https://docs.python.org/3/using/cmdline.html#cmdoption-u), I added it into the `const api = spawn('python', ['-u', '-m', 'qiime_studio']` and everything is coming out exactly how it should without any tweaks to the code, though still being fed through the error portion, which seems to be a gevent issue.